BannedScript/banScript.py

```diff
@@ -0,0 +1,60 @@
+import yaml
+import pymongo
+import requests
+
+# Load YAML file
+with open('./config/config.yml', 'r') as file:
+    config = yaml.safe_load(file)
+
+authToken = input("Please provide your token: ")
+username = input("Please enter username to whom you want to ban: ")
+
+dbUrl = config['database']['url']
+dbName = config['database']['name']
+dbCollection = config['database']['collection']
+
+# Connect to the MongoDB server
+client = pymongo.MongoClient(dbUrl)
+
+# Access a specific database
+db = client[dbName]
+
+# Access a specific collection (similar to a table in SQL)
+collection = db[dbCollection]
+
+uri = config['API']['uri']
+
+headers = {
+    "Accept": "application/json, text/plain, */*",
+    "Content-Type": "application/json",
+    "authorization": "Bearer"+authToken
+}
+
+user = collection.find_one({"username": username})
+
+if user is None:
+    print("No such username exist")
+
+else :
+    collection.update_one(
+        {"_id": user['_id']},
+        {"$set": {"isBanned": True}}
+    )
+    payload = {
```

Comment:
payload = {
    "token": user['fcmToken'],
    "eventType": "User Trigger",
    "userid": user['_id'],
    "title": "Rule-Breaking? Not Cool!",
    "content": "Hey, we love your enthusiasm, but not your recent posts. You're banned for rule-breaking. Let’s keep it clean next time!",
    "notificationBody": "You’ve been banned for breaking our rules. Take a break and come back when you’re ready to behave!",
    "notificationTitle": "Ban Hammer Time!"
}

use this